modules/claims_api/app/swagger/claims_api/v1/swagger.json

```diff
@@ -358,7 +430,7 @@
                   "errors": [
                     {
                       "title": "Resource not found",
-                      "detail": "Claims not found",
+                      "detail": "The BGS server did not find the resource.",
```

Comment:
I thought we removed the callout for "BGS server" in the tooling that returned this before? is that not the case?

Reply:
Yes, you are correct, I forgot about that. I will track down from where this came.

Reply:
[it's from here](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/spec/lib/claims_api/error/soap_error_handler_spec.rb). This spec defines a controller and assigns the routes and then mocks the response from the controller, so we're essentially calling a fully mocked spec.

Reply:
yep, I already updated that spec!